<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mp3window</class>
 <widget class="QWidget" name="mp3window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="volumeUpButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>138</width>
     <height>98</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VOL+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeDownButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>101</y>
     <width>138</width>
     <height>98</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VOL-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infoEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1</y>
     <width>338</width>
     <height>98</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>101</y>
     <width>113</width>
     <height>98</height>
    </rect>
   </property>
   <property name="text">
    <string>Előző 
 szám</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>101</y>
     <width>113</width>
     <height>98</height>
    </rect>
   </property>
   <property name="text">
    <string>Lejátszás
/
Megállítás</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>101</y>
     <width>113</width>
     <height>98</height>
    </rect>
   </property>
   <property name="text">
    <string>Következő 
szám</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>200</y>
     <width>478</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KILÉPÉS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
